DFC.ServiceTaxonomy.UnitTests/Neo4j/Extensions/DriverExtensionsTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using DFC.ServiceTaxonomy.Neo4j.Extensions;
+using DFC.ServiceTaxonomy.Neo4j.Models;
+using FakeItEasy;
+using Neo4j.Driver;
+using Xunit;
+
+namespace DFC.ServiceTaxonomy.UnitTests.Neo4j.Extensions
+{
+    public class DriverExtensionsTests
+    {
+        [Fact]
+        public void DriverExtensions_WhenPassedCollection_ReturnsPrimaryDriver()
+        {
+            var listOfDrivers = new List<NeoDriver>() {
+                new NeoDriver("Primary", A.Fake<IDriver>(), "http://my-primary-driver"),
+                new NeoDriver("Secondary", A.Fake<IDriver>(), "http://my-secondary-driver"),
+                new NeoDriver("Secondary", A.Fake<IDriver>(), "http://my-secondary-2-driver"),
+            };
+
+            var primaryDriver = listOfDrivers.PrimaryDriver();
+
+            Assert.NotNull(primaryDriver);
```

Comment:
Could this check that the returned driver _is_ the primary driver (in addition to testing that _a_ driver is returned)?

Reply:
Driver Extensions being removed, but yeah valid point :)